test/testdata/packager/export_all/foo_bar_baz/foo_bar_baz.rb

```diff
@@ -2,5 +2,9 @@
 
 module Foo::Bar::Baz
   class Quux
+    extend T::Sig
+
+    sig { void }
+    def example; end
```

Comment:
Out of curiosity, why is this change sufficient to test the behavior?

Should there not also be some sort of error assertion that says in one case, something is not visible, and in another case, it is?

Reply:
This change adds "behavior" (i.e. methods) to the body of the class; the class was previously non-behavior-defining. The fix here is relevant when 1. there is a subpackage that 2. contains non-behavior-defining outer namespaces with 3. behavior-defining inner constants.